Makefile

```
@@ -25,7 +25,7 @@ ppx_inline_test \
 ppxlib \
 ctypes \
 "utop>=2.6.0" \
-"melange=1.0.0"
+"melange=3.0.0-414"
```

```suggestion
"melange>=3.0.0"
```

to easily allow `-51`, we'll add a bound if this becomes necessary